person.rb

```diff
@@ -1,13 +1,8 @@
-require_relative 'nameable'
+class Person
+  attr_accessor :name, :rentals
 
-class Person < Nameable
-  def initialize(age, name)
-    super()
-    @age = age
+  def initialize(name)
     @name = name
-  end
-
-  def correct_name
-    @name
+    @rentals = []
   end
 end
```

Comment:
- [ ] The same as the previous requirement, please add the `add_rentals` custom method for the Person class as well, but with the correct modification in the parameter. Instead of person and date, here it'll be book and date because we're in the Person class, not the Book class.